src/components/Proposal/Voting/VoteButtons.tsx

```diff
@@ -91,20 +89,20 @@ class VoteButtons extends React.Component<IProps, IState> {
                             ;
 
     /**
-     * only invoked when votingDisabled
-     * @param vote
+     * only used when votingDisabled
      */
-    const tipContent = (vote: IProposalOutcome|undefined): string =>
+    const tipContent = 
       ((currentVote === IProposalOutcome.Pass) || (currentVote === IProposalOutcome.Fail)) ?
         "Can't change your vote" :
         (currentAccountState && currentAccountState.reputation.eq(new BN(0))) ?
           "Voting requires reputation in " + dao.name :
-          proposal.stage === IProposalStage.ExpiredInQueue || (proposal.stage === IProposalStage.Boosted && expired) || (proposal.stage === IProposalStage.QuietEndingPeriod && expired)  || (proposal.stage === IProposalStage.Queued && expired) ?
+          proposal.stage === IProposalStage.ExpiredInQueue ||
+              (proposal.stage === IProposalStage.Boosted && expired) ||
+              (proposal.stage === IProposalStage.QuietEndingPeriod && expired)  ||
+              (proposal.stage === IProposalStage.Queued && expired) ?
             "Can't vote on expired proposals" :
             proposal.stage === IProposalStage.Executed ?
-              "Can't vote on executed proposals" :
-              `Vote ${vote === IProposalOutcome.Pass ? "for" : "against"}`
```

Comment:
im confused, why did you get rid of this one?

Reply:
That was a really weird piece of code.  Note that `tipContent` was a function before, taking the parameter `vote`.  Where was `vote` supposed to come from, and what did it represent?  I could find no answers so I removed it.

Reply:
looks pretty obvious in the code, you pass in whether it is a pass button or fail button you are mousing over and then the tip says Vote For or Vote Against. im not sure when that funcionality was removed but it used to exist. dont know if we care, no one has mentioned it.

Reply:
this text is only being used when voting is disabled, so the case you describe can't happen.